src/storage-client/src/controller.rs

```diff
@@ -2298,6 +2304,25 @@ where
                     frontier.clone(),
                 )]));
             }
+
+            // Check if the collection is for a Webhook source, unregister if so.
```

Comment:
nit: would prefer this were closer to where other source dropping code runs, e.g. https://github.com/MaterializeInc/materialize/pull/20472/files#diff-3154d32c4d0818b6513328628453395000c38bd1238bd1fa9e96b796201b38a6R2289

Reply:
Updated!